console/src/main/scripts/plugins/metrics/ts/app-details/appServerDeploymentsDetails.ts

```diff
@@ -57,7 +58,8 @@ module HawkularMetrics {
       $scope.vm = this;
       HawkularOps.init(this.NotificationsService);
 
-      this.startTimeStamp = +moment().subtract(1, 'hours');
+      this.resourceId = this.$routeParams.resourceId;
+      this.startTimeStamp = +moment().subtract(($routeParams.timeOffset || 3600000), 'milliseconds');
```

Comment:
We need to handle this default time period a bit better, so that if we change it in one place it gets changed everywhere. Nothing you need to do here just making a note of this. Ultimately, it should probably be user-definable anyway.